ViewModel/AlternativeComparisonReport/CreateNewAlternativeComparisonReportVM.cs

```diff
@@ -1,111 +1,212 @@
-﻿using ViewModel.Editors;
-using System;
+﻿using HEC.CS.Collections;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows;
+using ViewModel.Alternatives;
+using ViewModel.Editors;
+using ViewModel.Saving;
+using ViewModel.Utilities;
 
 namespace ViewModel.AlternativeComparisonReport
 {
     public class CreateNewAlternativeComparisonReportVM : BaseEditorVM
     {
-        private string _SelectedWithProjectPlan;
-        private ObservableCollection<Increment> _Increments = new ObservableCollection<Increment>();
-        //private ObservableCollection<string> _SelectedPlan1 = new ObservableCollection<string>();
-        //private ObservableCollection<string> _SelectedPlan2 = new ObservableCollection<string>();
+        private AlternativeComparisonReportElement _CurrentElement;
+        private bool _IsInEditMode;
 
-        //public ObservableCollection<string> SelectedPlan1
-        //{
-        //    get { return _SelectedPlan1; }
-        //    set { _SelectedPlan1 = value; NotifyPropertyChanged(); }
-        //}
-        //public ObservableCollection<string> SelectedPlan2
-        //{
-        //    get { return _SelectedPlan2; }
-        //    set { _SelectedPlan2 = value; NotifyPropertyChanged(); }
-        //}
-        public ObservableCollection<Increment> Increments
+        private int _SelectedIndex = 0;
+        private CustomObservableCollection<AlternativeComboItem> _ProjectAlternatives = new CustomObservableCollection<AlternativeComboItem>();
+        public CustomObservableCollection<AlternativeComboItem> ProjectAlternatives
         {
-            get { return _Increments; }
-            set { _Increments = value; NotifyPropertyChanged(); }
+            get { return _ProjectAlternatives; }
         }
-        public List<string> Plans { get; set; }
-        //change from string to "plan" once that object exists
-        //public string SelectedWithoutProjectPlan
-        //{
-        //    get;set;
-        //}
-        //public string SelectedWithProjectPlan
-        //{
-        //    get { return _SelectedWithProjectPlan; }
-        //    set { _SelectedWithProjectPlan = value; NotifyPropertyChanged(); }
-        //}
-        //public string SelectedWithProjectPlanSecondIncrement { get; set; }
+        public AlternativeComboItem SelectedWithoutProjectAlternative { get; set; }
```

Comment:
Same with this, since it has a public setter, and it's bound, you should do NotifyPropertyChanged()